ui/src/components/investor/OrderForm.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+import { Button, Form, Header, Message } from 'semantic-ui-react'
+
+import { DepositInfo } from './Investor'
+
+import './OrderForm.css'
+
+type Props = {
+    kind: 'bid' | 'offer';
```

Comment:
might want to make these global variables since they r hard coded types


`BID_KIND = 'bid'`
`OFFER_KIND = 'offer'`